openhtf/io/output/console_summary.py

```diff
@@ -0,0 +1,113 @@
+"""Module to display test summary on console."""
+import os
+import sys
+
+from openhtf import util
+from openhtf.io import test_record
+from openhtf.util import measurements as measurement
+
+
+class ConsoleSummary():
+  """Print test results with failure info on console. """
+
+  # pylint: disable=invalid-name
+  def __init__(self):
+    if os.name == 'posix':    #Linux and Mac
+      self.RED = '\033[91m'
+      self.GREEN = '\033[92m'
+      self.ORANGE = '\033[93m'
+      self.RESET = '\033[0m'
+      self.BOLD = '\033[1m'
+    else:
+      self.RED = ""
+      self.GREEN = ""
+      self.ORANGE = ""
+      self.RESET = ""
+      self.BOLD = ""
+
+    self.color_table = {
+        test_record.Outcome.PASS:self.GREEN,
+        test_record.Outcome.FAIL:self.RED,
+        test_record.Outcome.ERROR:self.ORANGE,
+        test_record.Outcome.TIMEOUT:self.ORANGE,
+    }
+  # pylint: enable=invalid-name
```

Comment:
This pylint disable should probably done only on the line where you want it applied, otherwise I think it's disabled for the whole block or until you explicitly re-enable it.